### ISSUE TITLE
@pdfme/generator not installing (maybe missing from npm)

### ISSUE THREAD

**Issue by CadeKritsch** [pdfme/pdfme#302]
### Describe the bug

I have @pdfme/generator v1.2.2 listed as one of my package.json dependencies, and it is causing the npm installation to stuck.  I even searched up the pdfme/generator package in the npm registry and it seems as though it was removed from the site.
![pdfme](https://github.com/pdfme/pdfme/assets/73767994/a11beb5a-cf51-434c-93ee-871b41e1a41f)

### To Reproduce

Add "@pdfme/generator": "~1.2.2" into package.json, try running npm install with any version of node and npm.  

### Expected behavior

A successful installation.

### Your Environment

```markdown
- pdfme package(@pdfme/generator or @pdfme/ui):@pdfme/generator
- pdfme version: 1.2.2
- Operating system: windows
- Node.js version or Browser name & version: node v 16, npm v 8
```

### Your Error Log

```shell
no error, the installation seemingly gets stuck
```

**Comment by hand-dot** [comment]
can you try bellow?

```
npm i @pdfme/generator@1.2.2
```

https://www.npmjs.com/package/@pdfme/generator/v/1.2.2

**Comment by pjm98** [comment]
This issue is getting out of hand.
I am trying to install the latest version of PDFME and npm just gets stuck trying to resolve the dependency tree.
Any news on this matter?

**Comment by CadeKritsch** [comment]
Reverting to version 1.2.2 fixed the issue for me.

**Comment by hand-dot** [comment]
@pjm98 
I'm sorry for the inconvenience. Can you update me on the current situation?

**Comment by jimseiwert** [comment]
Installing latest just sits here

```  npm i @pdfme/generator @pdfme/common --save
(#########⠂⠂⠂⠂⠂⠂⠂⠂⠂) ⠹ idealTree:rc-field-form: sill placeDep ROOT async-validator@3.5.2 OK for: form-render@2.3.5 want: ^3.5.1
```

**Comment by hand-dot** [comment]
@jimseiwert 
How did you think this log could solve the problem?





http://catb.org/~esr/faqs/smart-questions.html

**Comment by jimseiwert** [comment]
I wasn't sharing a log, I was just letting you know where it was hanging up, might want to reference your own link that you just deleted [not_losing](http://catb.org/~esr/faqs/smart-questions.html#not_losing)

**Comment by hand-dot** [comment]
@jimseiwert 
Oh, my apologies.
I'll respond calmly. Thank you.

**Comment by peteward** [comment]
Hi,
Please remember that this is open-source software. Minor sponsors aside, no one is getting paid for writing this software that is helping you or your business, so please try to respond constructively and courteously. If you want someone to give up their spare time to help resolve your issue then please provide as much information as possible to make it easier.

I've just switched to node 16 and tried to install pdfme generator 1.2.2 in a simple project with no problems.

If you are having this problem please can you share your full package.json? Also, have you tried installing another version of the generator and does this work? Even if you don't want another version for some reason at least can we work out if it's consistent with any version or just 1.2.2.

@jimseiwert, your message suggests that you're using version 3 of pdfme since `form-render` was added recently?

**Comment by dkastl** [comment]
Not sure this is related, but I also experienced that not all NPM packages were available for `3.0.0-beta.2`. Maybe there was an issue when publishing that version. And that was actually my main motivation to suggest automated builds: https://github.com/pdfme/pdfme/pull/306

So if you don't specifically require `1.2.2` it may be worth to try again.
